internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -152,113 +180,194 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIToolsToBedrockToolC
 	return nil
 }
 
-// OpenAIMessageToBedrockMessage converts openai ChatCompletion messages to aws bedrock messages
-func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) OpenAIMessageToBedrockMessage(openAIReq *openai.ChatCompletionRequest,
+// openAIMessageToBedrockMessageRoleUser converts openai user role message
+// returns a list of bedrock message and error
+// the tool content is appended to the list of bedrock message list if tool_call is in openai message
+func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIMessageToBedrockMessageRoleUser(
+	openAiMessage *openai.ChatCompletionMessageParamUnion,
+) ([]*awsbedrock.Message, error) {
+	message := openAiMessage.Value.(openai.ChatCompletionUserMessageParam)
+	bedrockMessages := make([]*awsbedrock.Message, 0)
+
+	if _, ok := message.Content.Value.(string); ok {
```

Comment:
you don't need to type assertion twice
```suggestion
	if v, ok := message.Content.Value.(string); ok {
```